pipelines/ApTemplate.yaml

```diff
@@ -21,6 +21,7 @@ tasks:
   makeWarp:
     class: lsst.pipe.tasks.makeCoaddTempExp.MakeWarpTask
     config:
+      connections.visitSummary: visitSummary
```

Comment:
Since this is the template-generation pipeline, it's worth asking whether we actually _want_ to use `finalVisitSummary` here. The usual objection that final stuff doesn't exist in the AP pipeline doesn't apply. However, I assume you verified that the `consolidateVisitSummary.connections.ConnectionsClass(config=consolidateVisitSummary).visitSummary.name ==
      makeWarp.connections.ConnectionsClass(config=makeWarp).visitSummary.name` contract still passes.

Reply:
Considering the rest of this pipeline fixed, I don't think you want to add the `updateVisitSummary` task in this pipeline to create `finalVisitSummary`, because it wouldn't actually have any updates to apply relative to what's in `visitSummary`.  It might be worth considering renaming the output of `consolidateVisitSummary` task here to `finalVisitSummary` instead, so that PipelineTasks that consume it could stick to their (new) defaults.  

You'd also certainly want it if you ever wanted to start running jointcal, doing a second round of PSF modeling, or doing full focal plane background subtraction - but I think if that was ever the case, you'd really just want to start using a DRP pipeline, even if we'd need to reorganize the package dependency structure to make that viable.  (I assume that's what we'll do to make templates in production anyway, even in year 1).

I have run `ap_verify` on `ap_verify_ci_hits2015` locally, so I assume that will have checked the contracts.